.github/actions/invalidate_master_cache/action.yml

```diff
@@ -0,0 +1,32 @@
+name: invalidate_npm_cache
```

Comment:
this action will work just after merge to master.
So we will have one version of cache on each merge to master and they last at most 1 week so we wont overuse the cache storage